librocksdb-sys/build.rs

```diff
@@ -308,7 +340,19 @@ fn try_to_find_and_link_lib(lib_name: &str) -> bool {
     false
 }
 
+fn cxx_standard() -> String {
+    env::var("ROCKSDB_CXX_STD").map_or("-std=c++11".to_owned(), |cxx_std| {
+        if !cxx_std.starts_with("-std=") {
+            format!("-std={}", cxx_std)
+        } else {
+            cxx_std
+        }
+    })
+}
+
 fn main() {
+    bindgen_rocksdb();
```

Comment:
Add one more function to bindgen for [this file](https://github.com/nervosnetwork/rust-rocksdb/blob/e731c9e202f6ec0e14fc38557a3e5f91580ab292/librocksdb-sys/src/patches.rs) too.

Reply:
All bundle in `bindings.rs`.